.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,69 @@
+name: "CI Tests"
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    services:
+      db:
+        image: postgres:11-alpine
+        ports: ['5432:5432']
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
I think we could improve the ident here `- uses: actions/checkout@v2`, it shouldn't aligned with the `steps`, what do you think :)

Reply:
Updated 🎉